### PR TITLE
Restricted Transfer

### PR DESCRIPTION
**Pull request by ayshptk**

### CODE REVIEW

File: src/modules/drop.ts [hunk 1]
```diff
@@ -1406,12 +1408,6 @@ class DropV1Module extends ModuleWithRoles<Drop> {
     return await this.sendTransaction("setMaxTotalSupply", [amount]);
   }
 
-  public async setRestrictedTransfer(
```
Comment (jakeloo) [inline src/modules/drop.ts:1409]: why is this nuked?

**Review by jakeloo — CHANGES_REQUESTED** [review]
also, docstring would be nice.

File: src/modules/pack.ts [hunk 2]
```diff
@@ -347,7 +347,9 @@ export class PackModule extends ModuleWithRoles<PackContract> {
   public async setRestrictedTransfer(restricted = false): Promise<void> {
     await this.sendTransaction("setRestrictedTransfer", [restricted]);
   }
-
+  public async isRestrictedTransfer(): Promise<boolean> {
```
Comment (atbe) [inline src/modules/pack.ts:350]: We should rename all instances of `isRestrictedTransfer` to `isTransferRestricted`.

File: src/modules/drop.ts (hunk 1 above)
Comment (atbe) [reply src/modules/drop.ts:1409]: moved to common interface, method name is still the same.

**Pull request merged**